composer.json

```diff
@@ -13,33 +13,32 @@
     "require": {
         "php": "^7.2|^8.0",
         "sulu/sulu": "^2.1 || 2.x-dev",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0 || ^7.0",
-        "symfony/messenger": "^4.4 || ^5.0 || ^6.0 || ^7.0",
+        "symfony/config": "^5.4 || ^6.0 || ^7.0",
```

Comment:
`4.4` was not tested and didn't even work because messenger initializiation failed with self reference issues